crates/nostr-mls/src/messages.rs

```diff
@@ -11,6 +11,7 @@
 //! The message content is encrypted using both MLS group keys and NIP-44 encryption.
 //! Message state is tracked to handle processing status and failure scenarios.
 
+use nostr::util::hex;
```

Comment:
Instead of using the `hex` crate, I've changed it to use the `nostr::util::hex` module